[2025-02-08T19:03:22.248+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:03:22.267+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-31T00:00:00+00:00 [queued]>
[2025-02-08T19:03:22.278+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-31T00:00:00+00:00 [queued]>
[2025-02-08T19:03:22.279+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:03:22.294+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-01-31 00:00:00+00:00
[2025-02-08T19:03:22.305+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2146) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:03:22.306+0000] {standard_task_runner.py:72} INFO - Started process 2171 to run task
[2025-02-08T19:03:22.305+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-01-31T00:00:00+00:00', '--job-id', '389', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmpwloox3uz']
[2025-02-08T19:03:22.308+0000] {standard_task_runner.py:105} INFO - Job 389: Subtask generate_customer_dim_data
[2025-02-08T19:03:22.353+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-31T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:03:22.438+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-31T00:00:00+00:00'
[2025-02-08T19:03:22.439+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:03:22.459+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T19:03:22.460+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:03:22.469+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:03:22.470+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2025-01-31T00:00:00+00:00, execution_date=20250131T000000, start_date=20250208T190322, end_date=20250208T190322
[2025-02-08T19:03:22.525+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:03:22.541+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:03:22.544+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:43:49.653+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:43:49.671+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-31T00:00:00+00:00 [queued]>
[2025-02-08T20:43:49.679+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-31T00:00:00+00:00 [queued]>
[2025-02-08T20:43:49.680+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:43:49.692+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-01-31 00:00:00+00:00
[2025-02-08T20:43:49.700+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=890) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:43:49.701+0000] {standard_task_runner.py:72} INFO - Started process 898 to run task
[2025-02-08T20:43:49.700+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-01-31T00:00:00+00:00', '--job-id', '521', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmp0il9ghyl']
[2025-02-08T20:43:49.702+0000] {standard_task_runner.py:105} INFO - Job 521: Subtask generate_customer_dim_data
[2025-02-08T20:43:49.743+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-31T00:00:00+00:00 [running]> on host 86932746817d
[2025-02-08T20:43:49.823+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-31T00:00:00+00:00'
[2025-02-08T20:43:49.824+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:43:49.843+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T20:43:49.843+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:43:49.851+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:43:49.852+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2025-01-31T00:00:00+00:00, execution_date=20250131T000000, start_date=20250208T204349, end_date=20250208T204349
[2025-02-08T20:43:49.875+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:43:49.892+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:43:49.894+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
